agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/AiComponentInstaller.java

```diff
@@ -152,22 +136,16 @@ private static void start(Instrumentation instrumentation) {
         }
         AppIdSupplier appIdSupplier = AppIdSupplier.INSTANCE;
 
-        TelemetryConfiguration configuration = TelemetryConfiguration.getActiveWithoutInitializingConfig();
-        TelemetryConfigurationFactory.INSTANCE.initialize(configuration, buildXmlConfiguration(config));
-        configuration.getContextInitializers().add(new SdkVersionContextInitializer());
-        configuration.getContextInitializers().add(new ResourceAttributesContextInitializer(config.customDimensions));
+        TelemetryClient telemetryClient = TelemetryClient.initActive(config.customDimensions, buildXmlConfiguration(config));
```

Comment:
should we track config.customDimensions as a feature in statsbeat? 
I wasn't aware of this config before.  if nobody uses it, we can consider removing it?

does telemetry processor enable users to add custom dimensions?